src/web/handlers/LongPressGestureHandler.ts

```diff
@@ -18,7 +18,7 @@ export default class LongPressGestureHandler extends GestureHandler {
   private startTime = 0;
   private previousTime = 0;
 
-  private activationTimeout: number | undefined;
+  private activationTimeout: NodeJS.Timeout | undefined;
```

Comment:
```suggestion
  private activationTimeout: ReturnType<typeof setTimeout> | undefined;
```

Reply:
Done in [3f244d2](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/3f244d27a397c68bf80abedc2c9ed1e450ebf534)